public/static/docs/command-reference/get.md

```diff
@@ -58,13 +58,17 @@ name.
   an existing directory is specified, then the output will be placed inside of
   it.
 
-- `--rev` - `url` is expected to represent a Git repository for this option to
+- `--rev` - `url` is expected to contain a Git repository for this option to
```

Comment:
I would say in this case we don't need to specify these "pre-req" at all - neither before or after :) May be something like "Obviously, a Git repository must be located at provide location (`url` arg)." at the end.

Reply:
Yeah I guess it's obvious. No need to add the "obviously" note though, doesn't sound friendly. Removed this prereq sentence.

Reply:
Applied the same for `dvc import`.